pingrid.py

```diff
@@ -190,6 +190,18 @@ def tile(da, tx, ty, tz, clipping=None, test_tile=False):
     return image_resp(im)
 
 
+def empty_tile(width: int = 256, height: int = 256):
+    # If tile size were hard-coded, this could be a constant instead
+    # of a function, but we're keeping open the option of changing
+    # tile size. Also, numpy arrays are mutable, and having a mutable
+    # global constant could lead to tricky bugs.
```

Comment:
The optimum probably depends on a lot of factors--network latency, network bandwidth, disk latency, disk bandwidth, server memory, server CPU, ... but I don't think there's a huge amount to be gained by spending time optimizing it. 256x256 seems to be the common choice for web Mercator maps. The reason I'm thinking about  other tile sizes is that if we move to the equirectangular projection, we might want to use rectangular tiles e.g. 256x128, since the world map isn't square in that projection.